axiom/cmd_txndata_transmit.c

```diff
@@ -52,12 +53,19 @@ char* nr_txndata_error_to_json(const nrtxn_t* txn) {
   user_attributes = nr_attributes_user_to_obj(txn->attributes,
                                               NR_ATTRIBUTE_DESTINATION_ERROR);
 
-  json = nr_error_to_daemon_json(txn->error, txn->name, agent_attributes,
-                                 user_attributes, txn->intrinsics,
-                                 txn->request_uri);
+  // add guid to aid error linking ui
+  // make copy of txn->intrisics to not cause it to be modified for other 
+  // potential uses during conversion to flatbuffer
+  intrinsics_attributes = nro_copy(txn->intrinsics);
```

Comment:
So the worry is that something inside `nr_error_to_daemon_json` will depend on `guid` in `txn->intrinsics` or that something outside of `nr_txndata_error_to_json` will depend on `guid` in `txn->intrinsics`? And the `nro_copy` strategy was used because `util_object.h` does not provide `nro_del_hash_string` API, right?

Reply:
I didn't want to modify `txn->instrinsics` for code that ran after `nr_error_to_daemon_json`.  Probably more out of caution of causing the least impact with these changes.  It seems most other agents send "guid" in intrinsics so perhaps it is being a bit cautious.